spec/factories/git_repositories.rb

```diff
@@ -1,5 +1,8 @@
+require 'faker'
+
 FactoryBot.define do
   factory :git_repository do
     sequence(:name) { |n| "git_repo_#{seq_padded_for_sorting(n)}" }
+    url { "https://#{Faker::Internet.domain_name}/#{Faker::Internet.domain_word}/#{Faker::Internet.domain_word}.git" }
```

Comment:
@bdunne you could put the require here, where it's used and will be lazy required when someone don't specify a url

Reply:
I was thinking that, but I'm going to want to use it in other factories, so it won't be here for long.

Reply:
Nitpicking here:  Could we technically get away with just one `/#{Faker::Internet.domain_word}` here?

I realize you are going with the GitHub REST route convention `/:user/:repo`, but technically that isn't required for the repo URLs, right?  We aren't parsing anything from the URL to generate a name or something (I hope...).